src/app/api/settings/validate-key/route.ts

```
@@ -0,0 +1,80 @@
+    let errorMessage = ''
+
+    try {
+      switch (provider) {
+        case 'replicate':
+          // Validate Replicate API key
+          const replicateResponse = await fetch('https://api.replicate.com/v1/account', {
+            headers: {
+              'Authorization': `Token ${apiKey}`,
+              'Content-Type': 'application/json'
+            }
+          })
```

Biome (2.1.2)

[error] Other switch clauses can erroneously access this declaration.
Wrap the declaration in a block to restrict its access to the switch clause.

The declaration is defined in this switch clause:

Safe fix: Wrap the declaration in a block.


(lint/correctness/noSwitchDeclarations)